crash when clicking send page, during sync

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x120cadb]

goroutine 16 [running]:
github.com/planetdecred/godcr/ui/page/components.(*AccountSelector).Layout.func2({{{0x0, 0x0}, {0x1d8, 0xf41cb}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/page/components/account_selector.go:157 +0x3b
gioui.org/layout.Flex.Layout({0xc3, 0x0, 0x0, 0x0}, {{{0x0, 0x0}, {0x1f8, 0xf41cb}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/layout/flex.go:99 +0xdb5
github.com/planetdecred/godcr/ui/decredmaterial.LinearLayout.Layout.func1.1.1.2.1.1.1({{{0x0, 0x0}, {0x1f8, 0xf41cb}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/decredmaterial/linearlayout.go:89 +0x85
gioui.org/layout.Direction.Layout(0x0, {{{0x0, 0x0}, {0x1f8, 0xf41cb}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/layout/layout.go:166 +0x162
github.com/planetdecred/godcr/ui/decredmaterial.LinearLayout.Layout.func1.1.1.2.1.1({{{0x1f8, 0x0}, {0x1f8, 0xf41cb}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/decredmaterial/linearlayout.go:88 +0xe5
gioui.org/layout.Inset.Layout({0x852, 0x0, 0x0, 0x0}, {{{0x1f8, 0x0}, {0x1f8, 0xf41cb}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/layout/layout.go:139 +0x3b6
github.com/planetdecred/godcr/ui/decredmaterial.LinearLayout.Layout.func1.1.1.2.1({{{0x210, 0x0}, {0x210, 0xf41e3}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/decredmaterial/linearlayout.go:86 +0x109
github.com/planetdecred/godcr/ui/decredmaterial.Border.Layout({{0x3d, 0x3d, 0x3d, 0xff}, {0x8, 0x8, 0x8, 0x8}, 0x40000000}, {{{0x210, ...}, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/decredmaterial/border.go:21 +0x98
github.com/planetdecred/godcr/ui/decredmaterial.LinearLayout.Layout.func1.1.1.2({{{0x210, 0x0}, {0x210, 0xf41e3}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/decredmaterial/linearlayout.go:84 +0x1d6
gioui.org/layout.Stack.Layout({0x0}, {{{0x0, 0x0}, {0x210, 0xf41e3}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/layout/stack.go:60 +0x8b6
github.com/planetdecred/godcr/ui/decredmaterial.LinearLayout.Layout.func1.1.1({{{0x0, 0x0}, {0x210, 0xf41e3}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/decredmaterial/linearlayout.go:47 +0x2aa
github.com/planetdecred/godcr/ui/decredmaterial.LinearLayout.Layout.func1.1({{{0x0, 0x0}, {0x210, 0xf41e3}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/decredmaterial/linearlayout.go:108 +0x238
gioui.org/layout.Inset.Layout({0x48b8, 0x0, 0x0, 0x0}, {{{0x0, 0x0}, {0x210, 0xf41e3}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/layout/layout.go:139 +0x3b6
github.com/planetdecred/godcr/ui/decredmaterial.LinearLayout.Layout.func1({{{0x0, 0x0}, {0x210, 0xf41e3}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/decredmaterial/linearlayout.go:44 +0x109
gioui.org/layout.Direction.Layout(0x0, {{{0x0, 0x0}, {0x210, 0xf41e3}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/layout/layout.go:166 +0x162
github.com/planetdecred/godcr/ui/decredmaterial.LinearLayout.Layout({0xfffffffffffffffe, 0xffffffffffffffff, 0x0, {0x0, 0x0, 0x0, 0x0}, 0x0, {{0x3d, 0x3d, ...}, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/decredmaterial/linearlayout.go:42 +0xe5
github.com/planetdecred/godcr/ui/page/components.(*AccountSelector).Layout(0xc00d916900, {0x1de6580, 0xc00077dda0}, {{{0x0, 0x0}, {0x210, 0xf41e3}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/page/components/account_selector.go:146 +0x3e5
github.com/planetdecred/godcr/ui/page/send.(*Page).layoutDesktop.func1.1({{{0x0, 0x0}, {0x210, 0xf41e3}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/page/send/layout.go:159 +0x87
gioui.org/layout.Flex.Layout({0x0, 0x0, 0x0, 0x0}, {{{0x0, 0x0}, {0x210, 0xf4208}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/layout/flex.go:99 +0xdb5
github.com/planetdecred/godcr/ui/page/send.(*Page).pageSections.func1.1({{{0x0, 0x0}, {0x210, 0xf4208}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/page/send/layout.go:277 +0x1f8
gioui.org/layout.Inset.Layout({0xbc, 0x0, 0xbb, 0x0}, {{{0x0, 0x0}, {0x210, 0xf4208}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/layout/layout.go:139 +0x3b6
github.com/planetdecred/godcr/ui/page/send.(*Page).pageSections.func1({{{0x0, 0x0}, {0x230, 0xf4228}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/page/send/layout.go:276 +0xca
gioui.org/layout.Stack.Layout({0x94}, {{{0x230, 0x0}, {0x230, 0xf4228}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/layout/stack.go:60 +0x8b6
github.com/planetdecred/godcr/ui/decredmaterial.Card.Layout.func1({{{0x230, 0x0}, {0x230, 0xf4228}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/decredmaterial/card.go:64 +0x18e
gioui.org/layout.Inset.Layout({0x30, 0x0, 0x30, 0x0}, {{{0x230, 0x0}, {0x230, 0xf4228}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/layout/layout.go:139 +0x3b6
github.com/planetdecred/godcr/ui/decredmaterial.Card.Layout({{0x0, 0x0, 0x0, 0x0}, {0x25, 0x25, 0x25, 0xff}, {0x12, 0x12, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/decredmaterial/card.go:63 +0x129
github.com/planetdecred/godcr/ui/page/send.(*Page).pageSections(0xc00336c000, {{{0x230, 0x0}, {0x230, 0xf4228}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/page/send/layout.go:275 +0x218
github.com/planetdecred/godcr/ui/page/send.(*Page).layoutDesktop.func1({{{0x230, 0x0}, {0x230, 0xf4228}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/page/send/layout.go:158 +0xd2
gioui.org/layout.Inset.Layout({0xbb, 0x0, 0xbc, 0x0}, {{{0x230, 0x0}, {0x230, 0xf4228}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/layout/layout.go:139 +0x3b6
github.com/planetdecred/godcr/ui/page/send.(*Page).layoutDesktop.func4.1.1.2.1.1({{{0x230, 0x0}, {0x230, 0xf4230}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/page/send/layout.go:183 +0x6b
gioui.org/layout.Inset.Layout({0xba, 0x0, 0x4778, 0x0}, {{{0x230, 0x0}, {0x230, 0xf4230}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/layout/layout.go:139 +0x3b6
github.com/planetdecred/godcr/ui/page/send.(*Page).layoutDesktop.func4.1.1.2.1({{{0x232, 0x0}, {0x232, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/page/send/layout.go:182 +0xc6
gioui.org/layout.(*List).Layout(0xc00decd550, {{{0x232, 0x0}, {0x232, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/layout/list.go:117 +0x17c
gioui.org/widget/material.ListStyle.Layout({0xc00decd500, {0xc00decd500, {0x40000000, 0x40000000, {0x0, 0x0, 0x0, 0x0}}, {0x41000000, 0x40c00000, ...}}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/widget/material/list.go:269 +0x20c
github.com/planetdecred/godcr/ui/decredmaterial.ListStyle.Layout(...)
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/decredmaterial/list_scroll.go:39
github.com/planetdecred/godcr/ui/page/send.(*Page).layoutDesktop.func4.1.1.2({{{0x0, 0x0}, {0x23c, 0x1cd}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/page/send/layout.go:181 +0x1c5
gioui.org/layout.Flex.Layout({0x0, 0x0, 0x0, 0x0}, {{{0x0, 0x0}, {0x23c, 0x1fd}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/layout/flex.go:99 +0xdb5
github.com/planetdecred/godcr/ui/page/send.(*Page).layoutDesktop.func4.1.1({{{0x0, 0x0}, {0x23c, 0x1fd}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/page/send/layout.go:174 +0x218
gioui.org/layout.Inset.Layout({0x12fa37d, 0x0, 0x0, 0x0}, {{{0x0, 0x0}, {0x23c, 0x1fd}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/layout/layout.go:139 +0x3b6
github.com/planetdecred/godcr/ui/page/components.UniformPadding({{{0x0, 0x0}, {0x26c, 0x22d}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/page/components/components.go:92 +0x197
github.com/planetdecred/godcr/ui/page/send.(*Page).layoutDesktop.func4.1({{{0x0, 0x0}, {0x26c, 0x22d}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/page/send/layout.go:173 +0xbd
gioui.org/layout.Stack.Layout({0xf0}, {{{0x26c, 0x22d}, {0x26c, 0x22d}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/layout/stack.go:78 +0x5dc
github.com/planetdecred/godcr/ui/page/send.(*Page).layoutDesktop.func4({{{0x26c, 0x22d}, {0x26c, 0x22d}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/page/send/layout.go:171 +0x18e
gioui.org/layout.Stack.Layout({0x0}, {{{0x0, 0x0}, {0x26c, 0x22d}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/layout/stack.go:78 +0x5dc
github.com/planetdecred/godcr/ui/page/send.(*Page).layoutDesktop(0xc00336c000, {{{0x0, 0x0}, {0x26c, 0x22d}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/page/send/layout.go:169 +0x3b4
github.com/planetdecred/godcr/ui/page/send.(*Page).Layout(0xc00336c000, {{{0x0, 0x0}, {0x26c, 0x22d}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/page/send/layout.go:152 +0x105
github.com/planetdecred/godcr/ui/page.(*MainPage).layoutDesktop.func1.1.1({{{0x0, 0x0}, {0x26c, 0x22d}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/page/main_page.go:667 +0x7d
gioui.org/layout.Flex.Layout({0x2a, 0x0, 0x0, 0x0}, {{{0x0, 0x0}, {0x320, 0x22d}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/layout/flex.go:99 +0xdb5
github.com/planetdecred/godcr/ui/decredmaterial.LinearLayout.Layout.func1.1.1.2.1.1.1({{{0x0, 0x0}, {0x320, 0x22d}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/decredmaterial/linearlayout.go:89 +0x85
gioui.org/layout.Direction.Layout(0x0, {{{0x0, 0x0}, {0x320, 0x22d}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/layout/layout.go:166 +0x162
github.com/planetdecred/godcr/ui/decredmaterial.LinearLayout.Layout.func1.1.1.2.1.1({{{0x320, 0x22d}, {0x320, 0x22d}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/decredmaterial/linearlayout.go:88 +0xe5
gioui.org/layout.Inset.Layout({0x26a120, 0xc0, 0x0, 0x0}, {{{0x320, 0x22d}, {0x320, 0x22d}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/layout/layout.go:139 +0x3b6
github.com/planetdecred/godcr/ui/decredmaterial.LinearLayout.Layout.func1.1.1.2.1({{{0x320, 0x22d}, {0x320, 0x22d}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/decredmaterial/linearlayout.go:86 +0x109
github.com/planetdecred/godcr/ui/decredmaterial.Border.Layout({{0x0, 0x0, 0x0, 0x0}, {0x0, 0x0, 0x0, 0x0}, 0x0}, {{{0x320, ...}, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/decredmaterial/border.go:21 +0x98
github.com/planetdecred/godcr/ui/decredmaterial.LinearLayout.Layout.func1.1.1.2({{{0x320, 0x22d}, {0x320, 0x22d}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/decredmaterial/linearlayout.go:84 +0x1d6
gioui.org/layout.Stack.Layout({0xd0}, {{{0x0, 0x0}, {0x320, 0x22d}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/layout/stack.go:60 +0x8b6
github.com/planetdecred/godcr/ui/decredmaterial.LinearLayout.Layout.func1.1.1({{{0x0, 0x0}, {0x320, 0x22d}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/decredmaterial/linearlayout.go:47 +0x2aa
github.com/planetdecred/godcr/ui/decredmaterial.LinearLayout.Layout.func1.1({{{0x0, 0x0}, {0x320, 0x22d}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/decredmaterial/linearlayout.go:108 +0x238
gioui.org/layout.Inset.Layout({0x2298e8, 0xc0, 0x24ce59a0, 0xc0ba61e5}, {{{0x0, 0x0}, {0x320, 0x22d}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/layout/layout.go:139 +0x3b6
github.com/planetdecred/godcr/ui/decredmaterial.LinearLayout.Layout.func1({{{0x0, 0x0}, {0x320, 0x22d}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/decredmaterial/linearlayout.go:44 +0x109
gioui.org/layout.Direction.Layout(0x0, {{{0x0, 0x0}, {0x320, 0x22d}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/layout/layout.go:166 +0x162
github.com/planetdecred/godcr/ui/decredmaterial.LinearLayout.Layout({0xfffffffffffffffe, 0xfffffffffffffffe, 0x0, {0x0, 0x0, 0x0, 0x0}, 0x0, {{0x0, 0x0, ...}, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/decredmaterial/linearlayout.go:42 +0xe5
github.com/planetdecred/godcr/ui/page.(*MainPage).layoutDesktop.func1.1({{{0x0, 0x0}, {0x320, 0x22d}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/page/main_page.go:661 +0x205
gioui.org/layout.Flex.Layout({0x5, 0x0, 0x0, 0x0}, {{{0x0, 0x0}, {0x320, 0x258}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/layout/flex.go:99 +0xdb5
github.com/planetdecred/godcr/ui/decredmaterial.LinearLayout.Layout.func1.1.1.2.1.1.1({{{0x0, 0x0}, {0x320, 0x258}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/decredmaterial/linearlayout.go:89 +0x85
gioui.org/layout.Direction.Layout(0x0, {{{0x0, 0x0}, {0x320, 0x258}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/layout/layout.go:166 +0x162
github.com/planetdecred/godcr/ui/decredmaterial.LinearLayout.Layout.func1.1.1.2.1.1({{{0x320, 0x258}, {0x320, 0x258}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/decredmaterial/linearlayout.go:88 +0xe5
gioui.org/layout.Inset.Layout({0x5d1ae8, 0xc0, 0xf6f9dd, 0x0}, {{{0x320, 0x258}, {0x320, 0x258}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/layout/layout.go:139 +0x3b6
github.com/planetdecred/godcr/ui/decredmaterial.LinearLayout.Layout.func1.1.1.2.1({{{0x320, 0x258}, {0x320, 0x258}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/decredmaterial/linearlayout.go:86 +0x109
github.com/planetdecred/godcr/ui/decredmaterial.Border.Layout({{0x0, 0x0, 0x0, 0x0}, {0x0, 0x0, 0x0, 0x0}, 0x0}, {{{0x320, ...}, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/decredmaterial/border.go:21 +0x98
github.com/planetdecred/godcr/ui/decredmaterial.LinearLayout.Layout.func1.1.1.2({{{0x320, 0x258}, {0x320, 0x258}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/decredmaterial/linearlayout.go:84 +0x1d6
gioui.org/layout.Stack.Layout({0x0}, {{{0x0, 0x0}, {0x320, 0x258}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/layout/stack.go:60 +0x8b6
github.com/planetdecred/godcr/ui/decredmaterial.LinearLayout.Layout.func1.1.1({{{0x0, 0x0}, {0x320, 0x258}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/decredmaterial/linearlayout.go:47 +0x2aa
github.com/planetdecred/godcr/ui/decredmaterial.LinearLayout.Layout.func1.1({{{0x0, 0x0}, {0x320, 0x258}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/decredmaterial/linearlayout.go:108 +0x238
gioui.org/layout.Inset.Layout({0x475ae7, 0x0, 0xf5314c, 0x0}, {{{0x0, 0x0}, {0x320, 0x258}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/layout/layout.go:139 +0x3b6
github.com/planetdecred/godcr/ui/decredmaterial.LinearLayout.Layout.func1({{{0x0, 0x0}, {0x320, 0x258}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/decredmaterial/linearlayout.go:44 +0x109
gioui.org/layout.Direction.Layout(0x0, {{{0x0, 0x0}, {0x320, 0x258}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/layout/layout.go:166 +0x162
github.com/planetdecred/godcr/ui/decredmaterial.LinearLayout.Layout({0xfffffffffffffffe, 0xfffffffffffffffe, 0x1, {0x0, 0x0, 0x0, 0x0}, 0x0, {{0x0, 0x0, ...}, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/decredmaterial/linearlayout.go:42 +0xe5
github.com/planetdecred/godcr/ui/page.(*MainPage).layoutDesktop.func1({{{0x0, 0x0}, {0x320, 0x258}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/page/main_page.go:654 +0x20b
gioui.org/layout.Stack.Layout({0x90}, {{{0x0, 0x0}, {0x320, 0x258}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/layout/stack.go:78 +0x5dc
github.com/planetdecred/godcr/ui/page.(*MainPage).layoutDesktop(0xc0059b9200, {{{0x0, 0x0}, {0x320, 0x258}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/page/main_page.go:648 +0x12b
github.com/planetdecred/godcr/ui/page.(*MainPage).Layout(0xc0059b9200, {{{0x0, 0x0}, {0x320, 0x258}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/page/main_page.go:644 +0x105
github.com/planetdecred/godcr/ui.(*Window).prepareToDisplayUI.func2({{{0x0, 0x0}, {0x320, 0x258}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, 0x3d519d211d, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/window.go:226 +0xd9
gioui.org/layout.Stack.Layout({0x20}, {{{0x320, 0x258}, {0x320, 0x258}}, {0x3f800000, 0x3f800000}, {0x1d95ec0, 0xc0002298e8}, {0xc0ba61e524ce59a0, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.0.0-20220601100144-a896a467ecae/layout/stack.go:60 +0x8b6
github.com/planetdecred/godcr/ui.(*Window).prepareToDisplayUI(0xc00020bd60, {{0xc0ba61e524ce59a0, 0x3d519d211d, 0x2efa300}, {0x3f800000, 0x3f800000}, {0x320, 0x258}, {0x0, 0x0, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/window.go:241 +0x42d
github.com/planetdecred/godcr/ui.(*Window).handleFrameEvent(0xc00020bd60, {{0xc0ba61e524ce59a0, 0x3d519d211d, 0x2efa300}, {0x3f800000, 0x3f800000}, {0x320, 0x258}, {0x0, 0x0, ...}, ...})
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/window.go:182 +0x227
github.com/planetdecred/godcr/ui.(*Window).HandleEvents(0xc00020bd60)
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/ui/window.go:148 +0x156
main.main.func2()
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/main.go:80 +0x26
created by main.main
        /home/kennedy/Projects/src/github.com/planetdecred/godcr/main.go:79 +0x4d2
exit status 2
```

--- a/ui/modal/create_password_modal.go
+++ b/ui/modal/create_password_modal.go
@@ -103,6 +103,11 @@ func (cm *CreatePasswordModal) EnableConfirmPassword(enable bool) *CreatePasswor
 	return cm
 }
 
+func (cm *CreatePasswordModal) NameHint(hint string) *CreatePasswordModal {
+	cm.walletName.Hint = hint
+	return cm
+}
+
 func (cm *CreatePasswordModal) PasswordHint(hint string) *CreatePasswordModal {
 	cm.passwordEditor.Hint = hint
 	return cm

--- a/ui/page/components/nav_drawer.go
+++ b/ui/page/components/nav_drawer.go
@@ -51,8 +51,16 @@ func (nd *NavDrawer) LayoutNavDrawer(gtx layout.Context) layout.Dimensions {
 		layout.Rigid(func(gtx C) D {
 			list := layout.List{Axis: layout.Vertical}
 			return list.Layout(gtx, len(nd.DrawerNavItems), func(gtx C, i int) D {
-
+				mGtx := gtx
 				background := nd.Theme.Color.Surface
+
+				if nd.WL.SelectedWallet.Wallet.IsWatchingOnlyWallet() && (nd.DrawerNavItems[i].PageID == values.String(values.StrSend) ||
+					nd.DrawerNavItems[i].PageID == values.String(values.StrReceive) ||
+					nd.DrawerNavItems[i].PageID == values.String(values.StrAccountMixer)) {
+					background = decredmaterial.Disabled(nd.Theme.Color.Gray5)
+					mGtx = gtx.Disabled()
+				}
+
 				if nd.DrawerNavItems[i].PageID == nd.CurrentPage {
 					background = nd.Theme.Color.Gray5
 				}
@@ -65,7 +73,7 @@ func (nd *NavDrawer) LayoutNavDrawer(gtx layout.Context) layout.Dimensions {
 					Direction:   nd.direction,
 					Background:  background,
 					Clickable:   nd.DrawerNavItems[i].Clickable,
-				}.Layout(gtx,
+				}.Layout(mGtx,
 					layout.Rigid(func(gtx C) D {
 						img := nd.DrawerNavItems[i].ImageInactive
 

--- a/ui/page/main_page.go
+++ b/ui/page/main_page.go
@@ -640,7 +640,7 @@ func (mp *MainPage) OnNavigatedFrom() {
 // Layout draws the page UI components into the provided layout context
 // to be eventually drawn on screen.
 // Part of the load.Page interface.
-func (mp *MainPage) Layout(gtx layout.Context) layout.Dimensions {
+func (mp *MainPage) Layout(gtx C) D {
 	mp.Load.SetCurrentAppWidth(gtx.Constraints.Max.X)
 	if mp.Load.GetCurrentAppWidth() <= gtx.Dp(values.StartMobileView) {
 		return mp.layoutMobile(gtx)
@@ -648,7 +648,7 @@ func (mp *MainPage) Layout(gtx layout.Context) layout.Dimensions {
 	return mp.layoutDesktop(gtx)
 }
 
-func (mp *MainPage) layoutDesktop(gtx layout.Context) layout.Dimensions {
+func (mp *MainPage) layoutDesktop(gtx C) D {
 	return layout.Stack{}.Layout(gtx,
 		layout.Expanded(func(gtx C) D {
 			return decredmaterial.LinearLayout{
@@ -677,7 +677,7 @@ func (mp *MainPage) layoutDesktop(gtx layout.Context) layout.Dimensions {
 	)
 }
 
-func (mp *MainPage) layoutMobile(gtx layout.Context) layout.Dimensions {
+func (mp *MainPage) layoutMobile(gtx C) D {
 	return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
 		layout.Rigid(mp.LayoutTopBar),
 		layout.Flexed(1, func(gtx C) D {
@@ -685,7 +685,7 @@ func (mp *MainPage) layoutMobile(gtx layout.Context) layout.Dimensions {
 				layout.Expanded(func(gtx C) D {
 					currentPage := mp.CurrentPage()
 					if currentPage == nil {
-						return layout.Dimensions{}
+						return D{}
 					}
 					return currentPage.Layout(gtx)
 				}),
@@ -698,7 +698,7 @@ func (mp *MainPage) layoutMobile(gtx layout.Context) layout.Dimensions {
 	)
 }
 
-func (mp *MainPage) LayoutUSDBalance(gtx layout.Context) layout.Dimensions {
+func (mp *MainPage) LayoutUSDBalance(gtx C) D {
 	if !mp.usdExchangeSet {
 		return D{}
 	}
@@ -747,7 +747,7 @@ func (mp *MainPage) totalDCRBalance(gtx C) D {
 	return components.LayoutBalanceWithUnit(gtx, mp.Load, mp.totalBalance.String())
 }
 
-func (mp *MainPage) LayoutTopBar(gtx layout.Context) layout.Dimensions {
+func (mp *MainPage) LayoutTopBar(gtx C) D {
 	return decredmaterial.LinearLayout{
 		Width:       decredmaterial.MatchParent,
 		Height:      decredmaterial.WrapContent,

--- a/ui/page/settings_page.go
+++ b/ui/page/settings_page.go
@@ -104,7 +104,7 @@ func (pg *SettingsPage) Layout(gtx C) D {
 	return pg.layoutDesktop(gtx)
 }
 
-func (pg *SettingsPage) layoutDesktop(gtx layout.Context) layout.Dimensions {
+func (pg *SettingsPage) layoutDesktop(gtx C) D {
 	return layout.UniformInset(values.MarginPadding20).Layout(gtx, func(gtx C) D {
 		return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
 			layout.Rigid(pg.pageHeaderLayout),
@@ -115,7 +115,7 @@ func (pg *SettingsPage) layoutDesktop(gtx layout.Context) layout.Dimensions {
 	})
 }
 
-func (pg *SettingsPage) pageHeaderLayout(gtx layout.Context) layout.Dimensions {
+func (pg *SettingsPage) pageHeaderLayout(gtx C) layout.Dimensions {
 	return layout.Flex{Spacing: layout.SpaceBetween}.Layout(gtx,
 		layout.Flexed(1, func(gtx C) D {
 			return layout.W.Layout(gtx, func(gtx C) D {
@@ -133,14 +133,14 @@ func (pg *SettingsPage) pageHeaderLayout(gtx layout.Context) layout.Dimensions {
 	)
 }
 
-func (pg *SettingsPage) pageContentLayout(gtx layout.Context) layout.Dimensions {
+func (pg *SettingsPage) pageContentLayout(gtx C) D {
 	pageContent := []func(gtx C) D{
 		pg.general(),
 		pg.security(),
 		pg.info(),
 	}
 	gtx.Constraints.Min.X = gtx.Constraints.Max.X
-	return layout.Center.Layout(gtx, func(gtx layout.Context) layout.Dimensions {
+	return layout.Center.Layout(gtx, func(gtx C) D {
 		gtx.Constraints.Min.X = gtx.Dp(values.MarginPadding500)
 		gtx.Constraints.Max.X = gtx.Constraints.Min.X
 		gtx.Constraints.Min.Y = gtx.Constraints.Max.Y
@@ -150,8 +150,8 @@ func (pg *SettingsPage) pageContentLayout(gtx layout.Context) layout.Dimensions 
 	})
 }
 
-func (pg *SettingsPage) layoutMobile(gtx layout.Context) layout.Dimensions {
-	return layout.Dimensions{}
+func (pg *SettingsPage) layoutMobile(gtx C) D {
+	return D{}
 }
 
 func (pg *SettingsPage) settingLine(gtx C) D {

--- a/ui/page/wallet_settings_page.go
+++ b/ui/page/wallet_settings_page.go
@@ -282,7 +282,11 @@ func (pg *WalletSettingsPage) pageSections(gtx C, title string, body layout.Widg
 							}
 							if title == values.String(values.StrAccount) {
 								return layout.E.Layout(gtx, func(gtx C) D {
-									return pg.addAccount.Layout(gtx, pg.Theme.Icons.AddIcon.Layout24dp)
+									mGtx := gtx
+									if pg.WL.SelectedWallet.Wallet.IsWatchingOnlyWallet() {
+										mGtx = gtx.Disabled()
+									}
+									return pg.addAccount.Layout(mGtx, pg.Theme.Icons.AddIcon.Layout24dp)
 								})
 							}
 
@@ -699,6 +703,7 @@ func (pg *WalletSettingsPage) HandleUserInteractions() {
 		newPasswordModal := modal.NewCreatePasswordModal(pg.Load).
 			Title(values.String(values.StrCreateNewAccount)).
 			EnableName(true).
+			NameHint(values.String(values.StrAcctName)).
 			EnableConfirmPassword(false).
 			PasswordHint(values.String(values.StrSpendingPassword)).
 			PasswordCreated(func(accountName, password string, m *modal.CreatePasswordModal) bool {

--- a/ui/values/localizable/en.go
+++ b/ui/values/localizable/en.go
@@ -506,4 +506,5 @@ unlockWithPassword = "Unlock with password"
 "removeWalletInfo" = "%v Are you sure you want to remove %v %s%v? Enter the name of the wallet below to verify. %v"
 "confirmUmixedSpending" = "Confirm to allow spending from unmixed accounts"
 "ok" = "OK"
+"accountMixer" = "AccountMixer"
 `

--- a/ui/values/localizable/es.go
+++ b/ui/values/localizable/es.go
@@ -486,4 +486,5 @@ syncCompTime" = "Est. Sync completion time"
 "peer" = "Peer"
 "confirmUmixedSpending" = "Confirm to allow spending from unmixed accounts"
 "ok" = "OK"
+"accountMixer" = "AccountMixer"
 `

--- a/ui/values/localizable/fr.go
+++ b/ui/values/localizable/fr.go
@@ -463,4 +463,5 @@ const FR = `
 "peer" = "Peer"
 "confirmUmixedSpending" = "Confirm to allow spending from unmixed accounts"
 "ok" = "OK"
+"accountMixer" = "AccountMixer"
 `

--- a/ui/values/strings.go
+++ b/ui/values/strings.go
@@ -612,4 +612,5 @@ const (
 	StrPeer                            = "peer"
 	StrConfirmUmixedSpending           = "confirmUmixedSpending"
 	StrOK                              = "ok"
+	StrAccountMixer                    = "accountMixer"
 )

